src/test/java/api/loans/agetolost/ScheduledAgeToLostApiTest.java

```diff
@@ -0,0 +1,135 @@
+import static api.support.matchers.ItemMatchers.isClaimedReturned;
+import static api.support.matchers.JsonObjectMatcher.hasJsonPath;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.iterableWithSize;
+import static org.joda.time.DateTime.now;
+import static org.joda.time.DateTimeZone.UTC;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.hamcrest.Matcher;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import api.support.MultipleJsonRecords;
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.builders.ItemBuilder;
+import api.support.spring.SpringApiTest;
+import api.support.spring.clients.ScheduledJobClient;
+import io.vertx.core.json.JsonObject;
+import lombok.val;
+
+public class ScheduledAgeToLostApiTest extends SpringApiTest {
+  private IndividualResource overdueLoan;
+  private IndividualResource overdueItem;
+  @Autowired
+  private ScheduledJobClient scheduledAgeToLostClient;
+
+  public ScheduledAgeToLostApiTest() {
+    super(true, true);
+  }
+
+  @Before
+  public void activateLostItemFeePolicy() {
+    useLostItemPolicy(lostItemFeePoliciesFixture.ageToLostAfterOneMinute().getId());
+
+    checkOutItem();
+  }
+
+  @Test
+  public void shouldIgnoreOverdueLoansWhenItemIsClaimedReturned() {
+    claimItemReturnedFixture.claimItemReturned(overdueLoan.getId());
+
+    scheduledAgeToLostClient.triggerJob();
+
+    assertThat(itemsClient.get(overdueItem).getJson(), isClaimedReturned());
+  }
+
+  @Test
+  public void shouldNotAgeAnyItemsToLostIfNoIntervalDefined() {
+    val policy = lostItemFeePoliciesFixture.ageToLostAfterOneMinutePolicy()
+      .withName("Aged to lost disabled")
+      .withItemAgedToLostAfterOverdue(null);
+
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(policy).getId());
+
+    checkOutItem();
+
+    scheduledAgeToLostClient.triggerJob();
+
+    assertThat(itemsClient.get(overdueItem).getJson(), isCheckedOut());
+  }
+
+  @Test
+  public void shouldNotAgeItemToLostWhenNotOverdueByMoreThanIntervalYet() {
+    overdueItem = itemsFixture.basedUponNod();
+    overdueLoan = checkOutFixture.checkOutByBarcode(
+      new CheckOutByBarcodeRequestBuilder()
+        .forItem(overdueItem)
+        .at(servicePointsFixture.cd1())
+        .to(usersFixture.james())
+        .on(now(UTC)));
+
+    scheduledAgeToLostClient.triggerJob();
+
+    assertThat(itemsClient.get(overdueItem).getJson(), isCheckedOut());
+  }
+
+  @Test
+  public void shouldAgeItemToLostWhenOverdueByMoreThanInterval() {
+    scheduledAgeToLostClient.triggerJob();
+
+    assertThat(itemsClient.get(overdueItem).getJson(), isAgedToLost());
+    assertThat(getLoanActions(), hasAgedToLostAction());
+  }
```

Comment:
Could this positive scenario be moved to the top of the file, to help make it clear what the desired behaviour of this process is?

Reply:
sure, done.